shell.nix

```diff
@@ -0,0 +1,15 @@
+{ pkgs ? import
```

Comment:
What is this?

Reply:
That was meant to be in [another PR](https://github.com/stac-utils/stac-node-validator/pull/25). I just set that up to have an easily reproducible dev env.